core/scripts/keystone/main.go

```diff
@@ -15,13 +15,16 @@ type command interface {
 
 func main() {
 	commands := []command{
-		src.NewDeployContractsCommand(),
-		src.NewDeployJobSpecsCommand(),
-		src.NewGenerateCribClusterOverridesCommand(),
-		src.NewDeleteJobsCommand(),
+		src.NewProvisionKeystoneCommand(),
```

Comment:
For reviewers: This is the main entrypoint, CRIB scripts call into this command only to provision everything. All other commands below are essentially subcmds of this one